orchestration/templates/cron-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: encode-ingest
+spec:
+  suspend: {{ not .Values.cron.enable }}
+  schedule: {{ .Values.cron.schedule | quote }}
+  timezone: {{ .Values.cron.timezone | quote }}
+  concurrencyPolicy: Forbid
+  workflowSpec:
+    entrypoint: main
+    serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+    podGC:
+      strategy: OnWorkflowSuccess
+    parallelism: {{ .Values.maxParallelism }}
+    templates:
+      - name: main
+        steps:
+          - - name: run-main-template
+              templateRef:
+                name: ingest-encode-data
+                template: main
+              arguments:
+                parameters:
+                  - name: gcs-prefix
+                    value: '{{ template "argo.timestamp" }}'
```

Comment:
I based this off of https://github.com/DataBiosphere/clinvar-ingest/blob/master/orchestration/templates/cron-workflow.yaml

Reply:
Ah, wait: `argo.timestamp` won't be defined in this chart. The easiest thing to do would be to copy-paste the `_helpers.tpl` from clinvar. It'll eventually get bundled into a common chart that both can depend on...